Mage.Sets/src/mage/cards/c/CruelCelebrant.java

```diff
@@ -0,0 +1,51 @@
+
+package mage.cards.c;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.DiesThisOrAnotherCreatureOrPlaneswalkerTriggeredAbility;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.common.GainLifeEffect;
+import mage.abilities.effects.common.LoseLifeOpponentsEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.filter.common.FilterCreatureOrPlaneswalkerPermanent;
+import mage.filter.predicate.permanent.ControllerPredicate;
+
+/**
+ *
+ * @author Ketsuban
+ */
+public final class CruelCelebrant extends CardImpl {
+
+    private static final FilterCreatureOrPlaneswalkerPermanent filter = new FilterCreatureOrPlaneswalkerPermanent("creature or planeswalker you control");
+    static {
+        filter.add(new ControllerPredicate(TargetController.YOU));
+    }
+
+    public CruelCelebrant(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{W}{B}");
+
+        this.subtype.add(SubType.VAMPIRE);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(2);
+
+        // Whenever Cruel Celebrant or another creature or planeswalker you control dies, each opponent loses 1 life and you gain 1 life.
+        Ability ability = new DiesThisOrAnotherCreatureOrPlaneswalkerTriggeredAbility(new LoseLifeOpponentsEffect(1), false, filter);
+        ability.addEffect(new GainLifeEffect(1));
```

Comment:
you'll want to use the `.concatBy()` method to add "and" to this effect's text

Reply:
That code was copy-pasted from Blood Artist. I'll go ahead and fix that card while I'm at it.